.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,95 @@
+---
+
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug"]
+assignees:
+  - gordsport
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+
+  - type: textarea
+    id: bug
+    attributes:
+      label: Problem encountered
+      description: "A clear and concise description of what the bug is."
+    validations:
+      required: true
+
+  - type: textarea
+    id: observed
+    attributes:
+      label: Observed behaviour
+      description: "A clear and concise description of how the system is behaving."
+    validations:
+      required: true
+
+  - type: textarea
+    id: expected
+    attributes:
+      label: Expected behaviour
+      description: "A clear and concise description of what you expected to happen as per the AC or spec."
+    validations:
+      required: true
+
+  - type: textarea
+    id: reproduce
+    attributes:
+      label: Steps to reproduce
+      description: "Detail the steps to reproduce the behaviour."
+      value: |
+        1. Submit x transaction
+        2. With this payload
+        3. Query y API
+        4. Observe the error
+        ...
```

Comment:
Rest looks good to me, but I am not sure why we need this "..." ?
Are we supposed to point out the exact steps?
Or maybe we can use something like "please describe anything additional related to the observed behavior".

Reply:
Removed the ... and added some more info on what content is expected:
[d8b075a](https://github.com/vegaprotocol/vega/pull/8244/commits/d8b075a30807f0be5bbb5defe3ff485f038f18c9)